sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/CatalystTypeConvertersSuite.scala

```diff
@@ -148,7 +148,7 @@ class CatalystTypeConvertersSuite extends SparkFunSuite with SQLHelper {
   }
 
   test("SPARK-24571: convert Char to String") {
-    val chr: Char = 'X'
+    val chr: Char = $"X"'
```

Comment:
Doesn’t look like this should have been changed?

Reply:
@martin-g could you have a look at these? does seem like the search and replace failed in a few cases

Reply:
Yes! It is possible that there are such regressions!
Please don't spend time on reviewing the PR until I change it from `Draft` to `Ready for review` !